ui/src/androidTest/java/com/xmartlabs/bigbang/ui/recyclerview/common/ListActivity.java

```diff
@@ -0,0 +1,27 @@
+package com.xmartlabs.bigbang.ui.recyclerview.common;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+
+public abstract class ListActivity<T extends RecyclerView.Adapter> extends Activity {
```

Comment:
I don't agree with you, I'm not testing the fragment, neither the activity, I'm only want to test a simple list! Each test should test only one thing, BTW: if I want to extend from the base I have to inject and other unnecessary stuff.